Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -91,6 +91,12 @@ def Matrix.vecMulLinear [Fintype m] (M : Matrix m n R) : (m → R) →ₗ[R] n 
   map_smul' _ _ := funext fun _ => smul_dotProduct _ _ _
 #align matrix.vec_mul_linear Matrix.vecMulLinear
 
+@[simp] theorem Matrix.vecMulLinear_apply' {R m n : Type _} [Semiring R] [Fintype m]
+    (M : Matrix m n R) (x : m → R) : M.vecMulLinear x = M.vecMul x := rfl
```

Comment:
I think probably we should just remove the `simps` so that this can get the unpriemd name. You also shouldn't need to redeclare the variables
```suggestion
@[simp] theorem Matrix.vecMulLinear_apply [Fintype m] (M : Matrix m n R) (x : m → R) 👍 
  M.vecMulLinear x = M.vecMul x := rfl
```